spec/components/application_record_card/view_preview.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ApplicationRecordCard
+  class ViewPreview < ViewComponent::Preview
+    def single_card
+      render_component(ApplicationRecordCard::View.new(record: mock_trainee))
+    end
+
+    def multiple_cards
+      render_component(ApplicationRecordCard::View.with_collection(mock_multiple_trainees))
+    end
+
+    def no_data
+      render_component(ApplicationRecordCard::View.new(record: Trainee.new(id: 1)))
+    end
+
+  private
+
+    def mock_trainee
+      Trainee.new(id: 1)
```

Comment:
I think this could use some more dummy data, the view preview for this is showing the error messages, so it ends up looking the same as the no_data version

Reply:
Good spot